packages/react-composites/src/composites/ChatComposite/FileDownloadErrorBar.tsx

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { IIconProps, MessageBar, MessageBarType, Stack } from '@fluentui/react';
+import React from 'react';
+
+/**
+ * @private
+ */
+export interface FileDownloadErrorBarProps {
+  /**callback to dismiss the download error message */
+  onDismissDownloadErrorMessage?: () => void;
```

Comment:
This prop should be required. In that case you also don't need to check that this prop exists in the impl below.

Reply:
done